nextjs-task-system/components/TableUsers.tsx

```
@@ -0,0 +1,130 @@
+  const handlePageChange = (newPage: number): void => {
+    setCurrentPage(newPage);
+  };
+
+  useEffect(() => {
+    filter(currentPage);
+  }, [currentPage]);
+
+  const filter = async (page: number): Promise<void> => {
+    setIsLoading(true);
+    const resp: UsersResponse | null = await getAllUsers(`page=${page}&limit=${limit}`);
+    if(resp && resp.totalItems) setTotalItems(resp.totalItems);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)